### DISCUSSION TITLE
Request to kindly released stable 22.03.0 version

### DISCUSSION THREAD

**Discussion by naveen1hot** [immortalwrt/immortalwrt#786] [General]
Hi I'm very much thankful for your efforts and helping to others. Kindly release new stable version ASAP

**Comment by 1715173329** [comment]
22.03 is buggy (firewall4 with nftables) and low performance (mainly caused by kernel 5.10), so ImmortalWrt won't maintain this branch.
21.02 is great and stable enough, please wait for 23.xx release.
